reverse.py

```diff
@@ -0,0 +1,40 @@
+from string import punctuation, digits
+
+def reversed_func(text):
```

Comment:
It is a good practive to use type hits across the python code. Although, python is a dynamically typed language, a lot of nitpicks and misscodes can be prevented by using type annotations in place. It is widely used practice among the projects.
Please read more about it
Here is one resource - https://dagster.io/blog/python-type-hinting

And please add type annotation to the variable `text` and specify return type annotation for your function